src/rules/braces_spacing.coffee

```diff
@@ -50,30 +53,41 @@ module.exports = class BracesSpacing
         loop
             totalDifference += difference
             nearestToken = tokenApi.peek(totalDifference)
-            continue if nearestToken[0] is 'OUTDENT' or nearestToken.generated?
+            continue if nearestToken?[0] is 'OUTDENT' or nearestToken?.generated?
             return nearestToken
 
     tokensOnSameLine: (firstToken, secondToken) ->
         firstToken[2].first_line is secondToken[2].first_line
 
-    getExpectedSpaces: (tokenApi, firstToken, secondToken) ->
+    tokenSetsMatch: (a, b) -> JSON.stringify(a) is JSON.stringify b
+
+    getExpectedSpaces: (tokenApi, tokens) ->
         config = tokenApi.config[@rule.name]
-        if firstToken[0] is '{' and secondToken[0] is '}'
+        mono = [ 'IDENTIFIER', @tokens...]
+        tokens = tokens
+            .map (token) -> token?[0]
+            .filter (token) -> token in mono
+
+        if @tokenSetsMatch tokens[0..1], @tokens
             config.empty_object_spaces ? config.spaces
+        else if @tokenSetsMatch mono, tokens.sort()
+            config.mono_object_spaces ? config.spaces
```

Comment:
Can you set the default under `empty_object_spaces: 0` above? probably `mono_object_spaces: 0`

Reply:
@UziTech Sure.

Reply:
It looks like it only makes linting fail but the tests were each running `npm run lint` after the tests.

I'll have to look into it more to figure out why linting fails.

Reply:
I suppose we probably shouldn't change the default since it seems that it will be a breaking change.